MediaBrowser.Providers/Subtitles/SubtitleManager.cs

```diff
@@ -196,6 +170,48 @@ public Task DownloadSubtitles(Video video, string subtitleId, CancellationToken
             }
         }
 
+        /// <inheritdoc />
+        public async Task UploadSubtitle(Video video, SubtitleResponse response)
+        {
+            var libraryOptions = BaseItem.LibraryManager.GetLibraryOptions(video);
+
+            await TrySaveSubtitle(video, libraryOptions, response);
+        }
+
+        private async Task TrySaveSubtitle(
```

Comment:
INFO : Better to let task return a success-code or object. Even if not used, it's better to know whether the task was executed successfully.